source/features/clean-notifications.tsx

```diff
@@ -0,0 +1,23 @@
+import './clean-notifications.css';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as pageDetect from '../libs/page-detect';
+
+function init(): void {
+	// Check if dropdown exists and the group by 'repositories' is applied
+	const dropdown = select('.js-check-all-container .dropdown');
+	if (dropdown && select('summary', dropdown)?.textContent?.trim().endsWith('Repository')) {
+		document.querySelector('.js-notifications-container')?.classList.add('rgh-clean-notifications');
+	}
+}
+
+features.add({
+	id: __filebasename,
+	description: 'Removes redundant user/repo from notifications when being grouped by repository',
+	screenshot: 'https://user-images.githubusercontent.com/37769974/80862883-0419bc80-8c96-11ea-84a7-15baf1eac0e6.png'
+}, {
+	include: [
+		pageDetect.isNotifications
+	],
+	init
+});
```

Comment:
Why add a JS file when adding the extra selector as suggested already works correctly? This feature should be CSS only. This is the whole CSS:

```css
/* Place discussion number before the title */
.js-notifications-group [id^="notification"] {
    flex-direction: row !important;
}

/* Hide duplicate repository URL */
.js-notifications-group [id^="notification"] > .d-flex > .f6 {
    font-size: 0 !important;
}

/* Size and position the discussion number */
.js-notifications-group [id^="notification"] > .d-flex > .f6 > span {
	display: block;
    font-size: 12px;
    margin-top: 2px; /* Align baseline with discussion title */
    margin-right: 8px;
    word-break: keep-all;
}
```

Reply:
<img width="539" alt="" src="https://user-images.githubusercontent.com/1402241/80864559-beec8f00-8c83-11ea-84d3-910cdf00fab7.png">

Reply:
> Why add a JS file

So that this feature can be disabled.

Reply:
> in case anything actually breaks.

That applies to any CSS feature, but we don't make every CSS feature disableable "just in case it breaks."

If it breaks _we_ should just disable it for everyone.